static/src/stylesheets/module/identity/_manage-account-tab.scss

```diff
@@ -12,6 +12,23 @@
         min-height: 200px;
         margin-top: $gs-baseline*4;
     }
+    /* Warning
+    ========================================================================== */
+    .manage-account-warning {
```

Comment:
depending on the length of `userDetails.alertText` you might be able to get away with using `.form__error` or `.form__info` for this.

Or, style-wise, if all you want is a multiline at the end, there's `<hr class="manage-account-divider">`.

I'm trying to move all our longer feedback messages to `.identity-forms-message` (you can see it live [here](https://profile.theguardian.com/verify-email?isSignupFlow=true)) BUT at the moment that's more of a full page sort of deal.